dm/simulator/internal/mcp/mcp_test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mcp
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/pingcap/tiflow/dm/pkg/log"
+)
+
+type testMCPSuite struct {
+	suite.Suite
+	mcp *ModificationCandidatePool
+}
+
+func (s *testMCPSuite) SetupSuite() {
+	assert.Nil(s.T(), log.InitLogger(&log.Config{}))
+}
+
+func (s *testMCPSuite) SetupTest() {
+	mcp := NewModificationCandidatePool()
+	for i := 0; i < 4096; i++ {
+		mcp.keyPool = append(mcp.keyPool, &UniqueKey{
+			RowID: i,
+			Value: map[string]interface{}{
+				"id": i,
+			},
+		})
+	}
+	s.mcp = mcp
+}
+
+func (s *testMCPSuite) TestMCPNextUK() {
+	for i := 0; i < 10; i++ {
+		theUK := s.mcp.NextUK()
```

Comment:
we can assert something, like `theUK` is not nil. In CI there's not a human reading the output.

Reply:
OK.  I'll refine the tests.